src/popup/pages/Home.tsx

```diff
@@ -68,8 +70,16 @@ export const Component = () => {
     return <SiteNotMonetized />
   }
 
+  if (hasAllSessionsInvalid) return <AllSessionsInvalid />
```

Comment:
Consistency with other conditions.

```suggestion
  if (hasAllSessionsInvalid) {
    return <AllSessionsInvalid />
  }
```